vector/src/main/java/im/vector/app/features/settings/devices/v2/othersessions/OtherSessionsViewModel.kt

```diff
@@ -55,7 +55,7 @@ class OtherSessionsViewModel @AssistedInject constructor(
         observeDevicesJob?.cancel()
         observeDevicesJob = getDeviceFullInfoListUseCase.execute(
                 filterType = currentFilter,
-                excludeCurrentDevice = true
+                excludeCurrentDevice = !initialState.includeCurrentSession
```

Comment:
Maybe we should always use the same pattern everywhere for this argument for consistency and to avoid errors? Either `excludeCurrentDevice` or `includeCurrentDevice`?

Reply:
Make sense, done.